.gitlab-ci.yml

```diff
@@ -1,3 +1,5 @@
+include: 'https://gitlab-templates.ddbuild.io/slack-notifier/v1/template.yml'
```

Comment:
is this OK to use on a public repo? (guaranteed to never contain anything sensitive, etc)

Reply:
discussed offline: there's nothing sensitive there, OK to use 👍